src/EduManagementLab.Api/Controllers/CoursesController.cs

```diff
@@ -165,6 +180,28 @@ public ActionResult<List<Course.Membership>> GetCourseMemberships(Guid courseId)
             }
         }
 
+        [HttpGet]
+        [Route("{courseId}/Membership/{userId}")]
```

Comment:
Rename the route to "{courseId}/Memberships/{userId}"
Its a collection of memberships.

Reply:
I change the Route to {courseId}/Memberships/{userId}